data/MIMIC_CXR.py

```diff
@@ -0,0 +1,132 @@
+from skimage.io import imread, imsave
+import numpy as np
+import os,sys,os.path
+import pandas as pd
+import torch
+import torch.utils.data
+
+
+class MIMICCXR_Dataset:
+    """
+
+    Dataset website here:
+    https://physionet.org/content/mimic-cxr-jpg/2.0.0/
+    """
+
+    def __init__(self, imgpath, csvpath, metacsvpath, views=["PA"], transform=None, data_aug=None,
+                 flat_dir=True, seed=0):
+
+        np.random.seed(seed)  # Reset the seed so all runs are the same.
+        self.MAXVAL = 255
+
+        self.pathologies = ["Enlarged Cardiomediastinum",
+                            "Cardiomegaly",
+                            "Lung Opacity",
+                            "Lung Lesion",
+                            "Edema",
+                            "Consolidation",
+                            "Pneumonia",
+                            "Atelectasis",
+                            "Pneumothorax",
+                            "Pleural Effusion",
+                            "Pleural Other",
+                            "Fracture",
+                            "Support Devices"]
```

Comment:
We can change this to a `__init__` parameter too, maybe.

Reply:
In fact, I recommend using Texar's `hyperparam` system: https://github.com/asyml/texar-pytorch/blob/master/texar/torch/data/data/paired_text_data.py#L226